app/frontend/src/components/MapSearch.tsx

```diff
@@ -112,36 +112,42 @@ export default function MapSearch({
 
```

Comment:
Probably nitpick - but can we rename the props `city` and `setCity` instead of `value` and `setValue`? That matches what it's called in the parent and makes it a bit more obvious I think what kinda value is setting.

Reply:
This was on purpose because it might be used for something else. In fact, it's "city" which is poorly named, but that's what it is called in the protobuf.